### DISCUSSION TITLE
creating variables

### DISCUSSION THREAD

**Discussion by snehlata08** [apache/superset#21472] [Ideas]
In grafana we can create multiple variables.A variable is a placeholder for a value. You can use variables in metric queries and in panel titles. So when you change the value, using the dropdown at the top of the dashboard, your panel’s metric queries will change to reflect the new value.

Variables allow you to create more interactive and dynamic dashboards. Instead of hard-coding things like server, application, and sensor names in your metric queries, you can use variables in their place. Variables are displayed as dropdown lists at the top of the dashboard. These dropdowns make it easy to change the data being displayed in your dashboard.

![image](https://user-images.githubusercontent.com/79985986/187900787-d8cb33b5-3b82-432a-a4c2-6cd0c17a0424.png)

But in superset there is no feature like this. It would be cool if superset introduces something like this to make the charts and dashboards more interactive.

**Comment by sfirke** [comment]
I believe the norm for this project is to reserve the Issues for bugs and to put feature requests in Discussions.  You might have better engagement with this as a Discussion.  It looks good and I would upvote.
